core/src/main/kotlin/org/jetbrains/research/testspark/core/utils/CommandLineRunner.kt

```diff
@@ -9,13 +10,13 @@ class CommandLineRunner {
         protected val log = KotlinLogging.logger {}
 
         /**
-         * Executes a command line process and returns the output as a string.
+         * Executes a command line process
          *
          * @param cmd The command line arguments as an ArrayList of strings.
-         * @return The output of the command line process as a string.
+         * @return A pair containing exit value and a string message containing execution results
```

Comment:
1. `exit value` -> `exit code [of the executed process]`
2. `message containing execution results` -> `message containing stdout and stderr of the executed process`.